backend/python/app/tests/graphql/mutations/test_story_mutation.py

```diff
@@ -0,0 +1,163 @@
+import re
+
+from ....models.story import Story
+
+
+def test_create_story(app, db, client):
+    result = client.execute(
+        """
+        mutation {
+            createStory(
+                contents: [
+                    "line 1", "line 2", "line 3",
+                ],
+                storyData: {
+                    title: "title test",
+                    description: "this should explain things",
+                    youtubeLink: "don't think we validate this yet",
+                    level: 10001
+                }
+            ) {
+                ok
+                story {
+                    id
+                    title
+                    description
+                    youtubeLink
+                    translatedLanguages
+                    level
+                    contents {
+                        id
+                        storyId
+                        lineIndex
+                        content
+                    }
+                }      
+            }
+        }
+    """
+    )
+
+    assert dict(result["data"]) == {
+        "createStory": {
+            "ok": True,
+            "story": {
+                "id": 1,
```

Comment:
@gaoxk 
> Is the database cleared after each test? Will the id always be 1?

@e-wai 
> Think so? I can change the scope on the db fixture if we don't want that but I think it drops and recreates after each test (function scope). Though, to be honest, it may speed up our tests a lot if we don't drop/recreate so much

Reply:
Much better, went 124.30s -> 13.22s